examples/audit/docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: "3"
+
+services:
+  flipt:
+    image: flipt/flipt:latest
+    command: ["./flipt", "--force-migrate"]
+    ports:
+      - "8080:8080"
```

Comment:
what if we mounted a local file for the audit log as a volume, then the user could also tail that file in another terminal to see it being written to? we could include that in the instructions?

like:

```yml
volumes:
      - "/tmp/flipt/audit.log:/var/opt/flipt/audit.log"
```

Reply:
My bad, this should prob be `/var/log/flipt/audit.log` instead of `/var/opt`. I copied and pasted from a previous example 😞 sorry